week2.Rmd

```diff
@@ -0,0 +1,138 @@
+
+Let's plot these two data points just to practice using ggplot:
+```{r Plot Mars temperatures with different albedos}
+plot_Mars_albedo <- (Mars_albedos %>% #we name our plot and call our data frame
+       ggplot() + #this is the plot function that will plot our data frame
+       aes(x = albedo, y=Mars_TempC) + #Let's assign x- and y-axis columns
+       scale_x_continuous(limits = c(0.15,0.70), breaks=seq (0.15,0.70,0.05)) + #we can define our x-axis range with limits and then breaks between those limits
+       geom_point() + #we want to see the two data points come up on our plot
+       labs (title = "Temperature of Mars for two different albedos", x = "albedo", y= "Temperature (degrees Celsius)") + #let's add some labels for clarity
+       geom_line()) + #this creates a line between our two data points so we can see the trend between them 
+       theme_bw() #this is an extra addition that helps our plot show up in a black and white color scale
+plot_Mars_albedo #re-enter the plot name and R will generate the plot!
+```
+Fantastic! We just made a plot to show how different albedos on Mars affect its temperature in degrees Celsius. You'll use this same code when it comes to plotting your answer in question 3.
+
+##Questions
+1) Explain what albedo and heat energy has to do with the difference in the feeling of wearing a black shirt versus a white shirt on a bright, hot summer day. 
+
+_______________________________________________________
+
+2) Solve for the Temperature of the earth using its modern albedo of 0.37. Report your answer in degrees Celsius. Hint: you can do this with a variable and list like we did last week. 
+
+________________________________________________________
+
+3) Follow the steps below to practice using coding language to investigate the temperature of the earth given varying albedo values: 
+   a) create a data frame which includes the relevant values and constants in the Stefan-Boltzmann law needed to calculate temperature.
+   b) add a range of albedos (), Stefan-Boltzmann constant (SBc),  and a calculated temperature column for the data frame. 
+   c) use the data frame to plot albedo versus temperature for the earth.
+   d) comment on the trend of the ggplot you created. Do the different albedos change the Temperature by much? Is the trend linear?
+Hint: look back to the example for all the code you'll need. Simply rename columns and data frames to reflect information for the earth instead of Mars!
+ 
+_________________________________________________________
+
+4) After completing questions 2 and 3, what do you think about the temperatures that you calculated? Do they seem too hot or too cold? If they weren't what you were expecting given the temerature of Earth today, what could be missing from the model that we considered in calculating the earth's temperature? 
+
+_________________________________________________________
```

Comment:
I really like the directions you give with the questions. You give enough guidance but not too much. That's always a tricky balance to find. Nice job!